### ISSUE TITLE
How many epochs will it take to get results similar to your pretrained model? Also is this Warning while training during epoch 3 expected?

### ISSUE THREAD

**Issue by DeusxVita** [xuxy09/RSC-Net#13]
Loading latest checkpoint [/content/drive/MyDrive/RSC_NET/RSC-Net-master/logs/RSC-Net/checkpoints/2023_05_08-22_37_47.pt]
 75% 3/4 [00:00<?, ?it/s]
Epoch 3:  32% 9620/30272 [00:01<?, ?it/s]
/usr/local/lib/python3.8/site-packages/torch/optim/lr_scheduler.py:117: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
100% 4/4 [00:01<00:00,  1.03s/it]

**Comment by Lzmlzmlzm2021** [comment]
The ./data folder and ./dataset folder in DropBox was deleted. Could you please share me the folder by links or any other ways. Thank you a lot.
